components/layout_2020/tests/floats.rs

```diff
@@ -547,9 +573,9 @@ fn check_floats_rule_3(placement: &FloatPlacement) {
             // Where the top of `b` should probably be 32px per Rule 3, but unless this distinction
             // is made the top of `b` could legally be 0px.
             if this_float.origin.block >= other_float.rect().max_block_position() ||
-                (this_float.info.size.block == Length::zero() &&
+                (this_float.info.size.block == Au::zero() &&
                     this_float.rect().max_block_position() < other_float.origin.block) ||
-                (this_float.info.size.block > Length::zero() &&
+                (this_float.info.size.block > Au::zero() &&
```

Comment:
I think you can do `is_zero()` instead of comparing to zero here.

Reply:
it seems we cannot directly use `is_zero()` here. While size is of type `Au`, block here is of a generic type `T`.